.gitattributes

```diff
@@ -1 +1,2 @@
+*.expected.out binary
```

Comment:
Just curious, do you have a use case for this?

Reply:
The problem is that on Windows, the default for Git is to CRLF-ify files that look like text on checkout. In particular, it CRLF-ifies the expected test output files which causes the tests to fail since Lean outputs LF newlines with this change. The .gitattribute turns this off.

Reply:
Or just `-text`, then Git will not touch anything